Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift

```diff
@@ -36,7 +36,7 @@ public final class FormToggleItemView: FormItemView<FormToggleItem> {
         switchControl.translatesAutoresizingMaskIntoConstraints = false
         switchControl.isOn = item.value
         switchControl.onTintColor = item.style.tintColor
-        switchControl.isAccessibilityElement = false
```

Comment:
This most probably is messing with the actual accessibility as it's set to false on purpose if you check out the init of the `FormToggleItemView`